NuGet.config

```diff
@@ -5,7 +5,8 @@
     <packageSources>
         <clear />
         <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-        <add key="ApplicationInsights-MyGet" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
+        <add key="ClrInstrumentationEngine-Dependency"
```

Comment:
Is this feed open to the public? Or will this prevent individuals from building locally? Also, is nuget.org upstreamable, so that we only have one feed?

Reply:
The CLRIE-Dependency feed is public (https://dev.azure.com/ms/CLRInstrumentationEngine/_packaging?_a=feed&feed=ClrInstrumentationEngine-Dependency), and public feeds currently don't support upstreaming.

Reply:
Thanks. So that means, there is no way to use a single feed in this repo?

Reply:
Correct, until we can upstream our public feed to nuget.org, we'll have to live with this setup.